### ISSUE TITLE
The new LICENSE file with GPLv2 text is not sufficient

### ISSUE THREAD

**Issue by ppisar** [Perl-Critic/Perl-Critic-StricterSubs#17]
The 75d0284c9b58060a43fb2a7661f3c2f34ea6eb95 commit replaces GPL+ or Artistic license with GPLv2+ license. This is not in line with how Perl is licensed <https://metacpan.org/release/SHAY/perl-5.34.1/source/README#L84> and with all the copyright declarations over this package:

~~~~
Copyright (c) 2007-2022 Jeffrey Ryan Thalhammer.
This program is free software; you can redistribute it and/or modify
it under the same terms as Perl itself.  The full text of this license
can be found in the LICENSE file included with this module.
~~~~

I recommend reverting the commit.

**Comment by ppisar** [comment]
The copyright statements are fine. They refer to LICENSE file and the file needs to comply to them.
What you need to do is restore the original content of the LICENSE file. That will fix this issue #17.

Then you need to edit an FSF address in that file to match the one found in <https://www.gnu.org/licenses/old-licenses/gpl-1.0.txt>. Compare how Perl did it  <https://github.com/Perl/perl5/commit/24e75b45b6cb0613493c8f70cdb8e911e656b275>. That will fix issue #2.

As a result, the LICENSE file will quote both GPLv1 and Artistic license texts and at the same time will contain the latest FSF address.

**Comment by petdance** [comment]
Done, thank you.
